src/service/emoji-log.ts

```diff
@@ -0,0 +1,29 @@
+import { EmojiEventResponder, RoleEvent } from '../runner';
+import { StandardOutput } from './output';
```

Comment:
```suggestion
import type { EmojiEventResponder, RoleEvent } from '../runner';
import type { StandardOutput } from './output';
```
こちらも `import type` しましょう